packages/live-update/CapawesomeCapacitorLiveUpdate.podspec

```
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '14.0'
   s.dependency 'Capacitor'
-  s.dependency 'Alamofire', '~> 5.9.0'
+  s.dependency 'Alamofire', '~> 5.10.2'
```

```suggestion
  s.dependency 'Alamofire', '~> 5.9'
```

Let's try this one. This should allow all versions until the next major one (including 5.10.2). This is what we actually want (see `Package.swift`). I thought this is already the case.

---

Hey, updated, tested and everything seems to work